woocommerce/google-listings-and-ads
# Get started page UX improvements: add FAQs card

### Changes proposed in this Pull Request:

Part of #1392, this PR adds the FAQs card to the new get started page.

Design: eQ9O4m2flzrcAiDMXkOW0m-fi-7350%3A121282

The latest copies and the links of the FAQ can be found in the `Marketing copy document` in this post: pcTzPl-i7-p2#design-requirements. Need to request access in order to see the content.

### Screenshots:

#### Desktop

<img width="1442" alt="Screenshot 2022-05-03 at 14 45 20" src="https://user-images.githubusercontent.com/914406/166413944-b3e69541-4b4b-4618-b98b-0d088f144420.png">

<img width="1345" alt="gla-get-started-faq-desktop-expanded" src="https://user-images.githubusercontent.com/914406/166412619-adef5854-3178-469e-8e28-43d5ef6da7a9.png">


#### Mobile

<img width="859" alt="Screenshot 2022-05-03 at 14 46 11" src="https://user-images.githubusercontent.com/914406/166414005-9b37bbd0-32b6-418a-affa-a5e7be94fb75.png">

<img width="549" alt="gla-get-started-faq-mobile-expanded" src="https://user-images.githubusercontent.com/914406/166412669-d27d3763-9d88-4e58-adbb-7ff58258f5a7.png">

#### Tablet

<img width="900" alt="Screenshot 2022-05-03 at 14 46 28" src="https://user-images.githubusercontent.com/914406/166414029-f32e3317-87e1-4e20-ade7-9cf6b3b2dcfb.png">

<img width="777" alt="gla-get-started-faq-tablet-expanded" src="https://user-images.githubusercontent.com/914406/166412672-1b096b32-f778-4958-aef9-3425f0cb43e2.png">


### Detailed test instructions:

1. Go to GLA's get started page: `https://domain.test/wp-admin/admin.php?page=wc-admin&path=%2Fgoogle%2Fstart`
2. Check the sixth card of the get started page, the UI should align with the design.
3. Use the mobile view from the dev tool, the text of the longer question shouldn't be overlapped with the caret down icon like the below image:
    - <img width="561" alt="Screenshot 2022-05-03 at 14 33 14" src="https://user-images.githubusercontent.com/914406/166412912-c1e8eb06-c470-4d62-9bbd-33d4ca38f672.png">

### Changelog entry

> Update - Add the FAQs card for UX improvements on get started page.
